gtest/src/manager.rs

```diff
@@ -439,23 +282,33 @@ impl ExtManager {
                 None => break,
             };
 
-            let mut actors = self.actors.borrow_mut();
-            let (actor, balance) = actors
-                .get_mut(&dispatch.destination())
-                .expect("Somehow message queue contains message for user");
-            let balance = *balance;
-
-            if actor.is_dormant() {
-                drop(actors);
-                self.process_dormant(balance, dispatch);
-            } else if let Some((data, code)) = actor.get_executable_actor_data() {
-                drop(actors);
-                self.process_normal(balance, data, code, dispatch);
-            } else if let Some(mock) = actor.take_mock() {
-                drop(actors);
-                self.process_mock(mock, dispatch);
-            } else {
-                unreachable!();
+            enum DispatchCase {
+                Dormant,
+                ExecutableData(ExecutableActorData, InstrumentedCode),
```

Comment:
same as the name of the method called further
```suggestion
                Normal(ExecutableActorData, InstrumentedCode),
```

Reply:
Renamed